components/Feature/VulnerabilitiesGrid/index.js

```diff
@@ -165,32 +166,32 @@ const VulnerabilitiesGrid = ({ resources, onUpdate }) => {
                               value: label
                             })
                           }
+                          aria-expanded={
+                            textinputs
+                              ? selectedVulnerabilities
+                              : undefined
```

Comment:
this could probably just be replaced with 'textinputs', and that should get value of true or false depending on whether textinputs exists. 
Unless I misunderstand what aria-expanded does